src/course-search/FacetListBase.jsx

```diff
@@ -69,6 +70,8 @@ const FacetListBase = ({
       items={renderItems()}
       title={title}
       isBold={isBold}
+      attribute={attribute}
+      {...props}
```

Comment:
I strongly disprefer passing props this way; it can make it very confusing for future people reading the code. If we know what the props are, let's pass them explicitly.

Reply:
I am not sure what is the better way in this particular scenario, can you please guide?

Reply:
Hmmm.... are these extra props all coming from algolia? If yes, maybe put a note in that we're passing in all the props from algolia.

Reply:
mostly are coming from Algolia and some are ours.